docs/transition.rst

```diff
@@ -63,7 +63,7 @@ suggesting alternatives even if they will stick around for a longer time.
 Do not use:
 
 -   `werkzeug.script`, replace it with custom scripts written with
-    `argparse` or something similar.
+    `argparse`, pallets/click or something similar.
```

Comment:
I'd just call it click. Right not it looks a bit like messed-up copy&paste to me ;)